crates/proto/Cargo.toml

```diff
@@ -40,8 +40,10 @@ dnssec-openssl = ["dnssec", "openssl"]
 dnssec-ring = ["dnssec", "ring"]
 dnssec = ["data-encoding"]
 testing = []
+tokio-io = []
 tokio-runtime = [ "tokio/rt-core", "tokio/udp", "tokio/tcp" ]
-default = ["tokio-runtime"]
+tokio-time = [ "tokio/time" ]
+default = ["tokio-runtime", "tokio-time", "tokio-io"]
```

Comment:
Should we just group all of these under `tokio-runtime`? I think your use case (at the moment until we support `async-std`) is the only use case where we want tokio disabled. And for that, I think tokio-runtime is going to be enough to go by.

So, my opinion, let's just add `tokio/time` to `tokio-runtime` (unless there's a reason not to), and use that for enabling/disabling Tokio.

Reply:
Yes, I agree with that. Do you think for the feature's name, tokio-compat is more appropriate thant tokio-runtime in this case?

Reply:
I am totally okay with tokio-runtime as well if you would like to keep it.

Reply:
We're going to end up with different runtime's supported, tokio will be one, async-std, and then bring your own (like Fuchsia). I was thinking `tokio-runtime` would parallel a future `async-std-runtime`. So I kinda like that name, and think it's more descriptive than `-compat`.

But it does shadow the internal name in tokio, so I'm open to other options.

Reply:
I see. I will leave it as "tokio-runtime" until there is more appropriate name.